sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample2_RbacScopeAssignment.md

```diff
@@ -0,0 +1,36 @@
+# Creating a Role Assignment for Specific Scopes
+
+By default role assignments apply to the global scope. It is also possible to be more specific by applying an assignment to the all keys scope or a specific `KeyVaultKey`.
+For information about interacting with a `KeyVaultKey` with a `KeyClient`, see the [Key Client README](../../Azure.Security.KeyVault.Keys/README.md) for links and instructions.
+
+## Assigning a Role to the All Keys Scope
+
+Let's assign a role to a service principal soo that it applies to all keys. To do this we'll need a role definition Id and a service principal object Id.
```

Comment:
"so"

Reply:
BTW, I highly recommend the "Visual Studio Spell Checker" extension for VS from "EWSoftware"  (see extensions manager), and "Code Spell Checker" from "Street Side Software" for Code. Paired with the "Docs Authoring Pack" for Code, they make authoring markdown better.

Reply:
I do use a spell checker, however apparently "soo"  is in the dictionary 😄